internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -881,6 +881,10 @@ func (c *ClusterManager) buildObservatoriumDexSecretResource() *k8sCoreV1.Secret
 
 func (c *ClusterManager) buildObservabilityRemoteWriteServiceAccountCredential(cluster *api.Cluster) *k8sCoreV1.Secret {
 	return &k8sCoreV1.Secret{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: k8sCoreV1.SchemeGroupVersion.Version,
```

Comment:
Unfortunately this is not fully correct still.
`APIVersion` contains the API Group and the API Version, not only the version. It is a confusing name.
In this case `k8sCoreV1.SchemeGroupVersion.String()` should be used instead.
See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#resources